README.md

```diff
@@ -56,3 +56,21 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+With Module 5 we have containerised the to-do app with Docker
+There are two environments you can run from the one dockerfile: Development using flask or Production using Gunicorn
+
+To run the development server, first build the image using this command:
+docker build --target development --tag todo-app-v0.3:dev .
+  
+Then to run the container, run this command:
+docker run --env-file ./.env -p 5100:5000 -it --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app-v0.3:dev
+
+To run the production server, first build the image using this command:
+docker build --target production --tag todo-app-v0.3:prod .
+
+Then to run the container, run this command:
+docker run --env-file ./.env -p 5100:8000 -it --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app-v0.3:prod
```

Comment:
We need the bind mount for the development container, because we want to see the changes that we make to the codebase be hot reloaded by flask (so we don't have to stop the container, rebuild the image, and then rerun).

However, for a production environment, we don't need it, because:
* The container is very likely not running from our machine, so the code changes  will arrive later
* If the production image is running from our local machine, having hot reload is  a bad idea: when developing we introduce bugs, unfinished features, etc, which would break production
* As it is now, gunicorn is not hot reloading anyway. You'd need the `--reload` argument for that